src/parser/syntax.ml

```diff
@@ -1277,6 +1284,7 @@ and explicit_list_items :
         | `End ->
             Parse_error.not_allowed token_after_list_item.location
               ~what:(Token.describe `End) ~in_what:(Token.describe token)
+              ~suggestion:"try to add '}' at the end of the list content."
```

Comment:
This brings some code duplication. It would look better if a new error was defined in Parse_error, with the suggestion, eg. `Parse_error.end_not_allowed`. The suggestion string could be changed not to be specific to lists.